src/screens/enter-password/enter-password.tsx

```diff
@@ -17,32 +22,31 @@ import { useEnterPasswordStyles } from './enter-password.styles';
 export const EnterPassword = () => {
   const styles = useEnterPasswordStyles();
   const { unlock } = useAppLock();
-
   const onSubmit = ({ password }: EnterPasswordFormValues) => unlock(password);
 
   return (
-    <View style={styles.root}>
-      <InsetSubstitute />
-
-      <ScreenContainer>
-        <Formik
-          initialValues={enterPasswordInitialValues}
-          validationSchema={enterPasswordValidationSchema}
-          onSubmit={onSubmit}>
-          {({ submitForm }) => (
-            <>
-              <Text>Your wallet have been locked</Text>
-              <Text>{'Enter a password to unlock it\n\n'}</Text>
-
-              <Text>Password</Text>
-              <FormPasswordInput name="password" />
-
-              <Button title="Unlock" onPress={submitForm} />
-              <EraseDataButton />
-            </>
-          )}
-        </Formik>
-      </ScreenContainer>
-    </View>
+    <ScreenContainer isFullScreenMode={true}>
+      <WelcomeHeader />
+      <View style={styles.imageView}>
+        <WelcomeLogo />
+      </View>
+      <Formik
+        initialValues={enterPasswordInitialValues}
+        validationSchema={enterPasswordValidationSchema}
+        onSubmit={onSubmit}>
+        {({ submitForm }) => (
+          <KeyboardAvoidingView style={styles.formikView} behavior={Platform.OS === 'ios' ? 'padding' : 'height'}>
```

Comment:
Should this component be only here?
Maybe this's a common keyboard behavior and `KeyboardAvoidingView` should be used in `app.tsx`?

Reply:
I think, not. Sometimes when we would like to hide some element like Erase Data when we open the keyboard.